PowerPointLabs/Test/UnitTest/SyncLab/SyncLabArtisticEffectTest.cs

```diff
@@ -4,48 +4,47 @@
 namespace Test.UnitTest.SyncLab
 {
     /// <summary>
-    /// DO NOT use PPT2013 to edit the source powerpoint.
-    /// PPT2013 exhibits strange behavior on ArtisticEffect.
-    /// ArtisticEffect is sometimes made permenant on pictures after saving the file.
-    /// i.e, image's artistic effect cannot be removed/undone
+    /// Be wary when using PPT2013 to edit the source powerpoint.
+    /// ArtisticEffect is sometimes made permenant on PPT2013
+    /// i.e, image's artistic effect cannot be removed/undone after the image is copied
     /// </summary>
     [TestClass]
-    public class SyncLabVisualEffectsTest : BaseSyncLabTest
+    public class SyncLabArtisticEffectTest : BaseSyncLabTest
     {
-        private const int OriginalReplaceEffectSlideNo = 4;
-        private const int OriginalSetEffectSlideNo = 6;
-        private const int DesiredReplaceEffectSlideNo = 7;
+        private const int OriginalSetEffectSlideNo = 4;
         private const int DesiredSetEffectSlideNo = 5;
+        private const int OriginalReplaceEffectSlideNo = 6;
+        private const int DesiredReplaceEffectSlideNo = 7;
         
         private const string SourceShape = "Picture 2";
         private const string SetArtisticEffect = "Picture 9";
         private const string ReplacedArtisticEffect = "Picture 10";
         
         protected override string GetTestingSlideName()
         {
-            return "SyncLab\\SyncLab_VisualEffects.pptx";
+            return "SyncLab\\SyncLab_ArtisticEffect.pptx";
         }
-        
+
         [TestMethod]
         [TestCategory("UT")]
-        public void TestReplaceArtisticEffect()
+        public void TestSetArtisticEffect()
```

Comment:
Would it be better to name it TestSyncArtisticEffect so that it is consistent with the other synclab tests?

Reply:
There's another method called TestSyncArtisticEffect, which does that.
ArtisticEffect is a little different from other formats as we have to replace all existing artistic effects on the destination object. This test focuses on the 'replacing' part.